span.go

```diff
@@ -90,39 +105,20 @@ type Span interface {
 
 	// Provides access to the Tracer that created this Span.
 	Tracer() Tracer
+
+	// DEPRECATED
+	LogEvent(event string)
```

Comment:
note that I've added these back, so while all Tracer impls will break, existing calling code should continue to function for the time being as long as it doesn't use MockTracer.